src/plugins/auto/src/plugin.cpp

```diff
@@ -435,7 +435,13 @@ std::shared_ptr<ov::ICompiledModel> Plugin::compile_model_impl(const std::string
         support_devices = filter_device_by_model(support_devices_by_property, model, load_config);
     } else {
         // AUTO / MULTI don't support caching explicitly, but can redirect this functionality to actual HW plugin
-        LOG_INFO_TAG("compile model with model path");
+        LOG_INFO_TAG("compile model with model path: %s", model_path.c_str());
+        auto m_model = get_core()->read_model(model_path, std::string{}, {});
+        bool is_LLM_model = ov::op::util::is_large_language_model(*m_model);
+        if (is_LLM_model) {
+            // disable cpu helper and runtime_fallback when the model is LLM, only one device need to compile model
+            disable_startup_runtime_fallback(load_config);
+        }
```

Comment:
consider below code lines to replace the changed lines here.
`auto m_model = model_path.empty() ? model : get_core()->read_model(mode_path, std::string{}, {});` and ` support_devices = filter_device_by_model(support_devices_by_property, m_model, load_config);`